### PR TITLE
Improved logging.

### PR DESCRIPTION
**Pull request by mathuin**
Existing logging is print-based and often fails to contain useful information.

Logging is now done with the logging module, with debug logs going to file and info to console.  Logging levels are raised to errors for testing to minimize spam.

### CODE REVIEW

File: pgd_splicer/ProcessPDBTask.py [hunk 1]
```diff
@@ -215,27 +219,28 @@ def process_pdb(data):
...
 
         # 1) parse with bioPython
-        data = parseWithBioPython(filename, data, chains_filter)
+        data = parseWithBioPython(code, data, chains_filter)
 
         # 2) Create/Get Protein and save values
         try:
             protein = ProteinModel.objects.get(code=code)
-            print '  Existing protein: ', code
+            logger.debug('protein exists')
```
Comment (pop) [inline pgd_splicer/ProcessPDBTask.py:241]: Should there be a bit more information here about _which_ protein exists or does not exist?

**Comment by pop** [comment]
I like moving away from [the python 2.X] printing as a form of debugging and toward a more standard logging system.
+1

File: pgd_splicer/ProcessPDBTask.py (hunk 1 above)
Comment (mathuin) [inline pgd_splicer/ProcessPDBTask.py:241]: On line 224, the logger is defined as coming from the `pdb['logger']` value, which was set on line 201 from the variable which is assigned on line 198.  The argument to the `getLogger(`) call is equal to the code of the protein.  That string is prefixed to all calls that logger makes, including the "protein exists" call.

The old logging output looked like this:

```
  Existing protein: 1XYZ
```

The new output looks like this:

```
1XYZ DEBUG    protein exists
```

Does that help?

Comment (pop) [inline pgd_splicer/ProcessPDBTask.py:241]: Ah, that does help a lot. LGTM

**Pull request merged**